omfile-hardened: Compile error

### Expected behavior

Success compile without error.

### Actual behavior

Compile fails with error.

### Steps to reproduce the behavior

`./configure --enable-omfile-hardened && make`

Error message is like below.
```
...
...

omfile-hardened.c: In function 'janitorChkDynaFiles':
omfile-hardened.c:1005:45: error: dereferencing pointer to incomplete type 'rsconf_t' {aka 'struct rsonf_s'}
 1005 |    pCache[i]->nInactive += runModConf->pConf->globals.janitorInterval;
      |                                             ^~
make[2]: *** [Makefile:564: omfile_hardened_la-omfile-hardened.lo] Error 1

...
...
```


### Environment
- rsyslog version: <!-- e.g. 8.35.0 - use "rsyslogd -v" to obtain the version number --> rsyslogd  8.2204.0.master (aka 2022.04) 
- platform: <!-- e.g. Ubuntu 16.04 --> Ubuntu 20.04
- for configuration questions/issues, include rsyslog.conf and included config files

<!-- Note: rsyslog documentation is available here:
  - current stable release: http://www.rsyslog.com/doc/v8-stable/
  - pre-release: http://www.rsyslog.com/doc/master/
-->

--- a/contrib/omfile-hardened/omfile-hardened.c
+++ b/contrib/omfile-hardened/omfile-hardened.c
@@ -73,6 +73,7 @@
 #include "cryprov.h"
 #include "parserif.h"
 #include "janitor.h"
+#include "rsconf.h"
 
 MODULE_TYPE_OUTPUT
 MODULE_TYPE_NOKEEP

